storage/redis/peer_store.go

```diff
@@ -43,14 +40,14 @@ func New(cfg Config) (storage.PeerStore, error) {
 	if cfg.Instance != 0 {
 		conn.Do("SELECT", cfg.Instance)
 	}
-	prefixedSeeder = addNameSpace(cfg.Namespace, "seeder")
-	prefixedLeecher = addNameSpace(cfg.Namespace, "leecher")
 
 	ps := &peerStore{
-		conn:         conn,
-		closed:       make(chan struct{}),
-		maxNumWant:   cfg.MaxNumWant,
-		peerLifetime: cfg.PeerLifetime,
+		conn:             conn,
+		closed:           make(chan struct{}),
+		maxNumWant:       cfg.MaxNumWant,
+		peerLifetime:     cfg.PeerLifetime,
+		seederKeyPrefix:  addNameSpace(cfg.KeyPrefix, "seeder"),
```

Comment:
You missed to rename the function name `addNameSpace`. you could rename it as `addKeyPrefix`